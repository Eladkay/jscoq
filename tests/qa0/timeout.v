Require Import Nat.

Axiom add_comm : forall n m , n + m = m + n.

Lemma zz x : x + 0 = x.

Timeout 1 repeat rewrite add_comm.
