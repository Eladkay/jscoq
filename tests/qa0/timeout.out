------------------------------------------------------------
[Error] Timeout!
------------------------------------------------------------
{"fname":["ToplevelInput"],"line_nb":1,"bol_pos":0,"line_nb_last":1,"bol_pos_last":0,"bp":0,"ep":29}
------------------------------------------------------------
[Error] Timeout!
------------------------------------------------------------
{"fname":["ToplevelInput"],"line_nb":1,"bol_pos":0,"line_nb_last":1,"bol_pos_last":0,"bp":0,"ep":29}
Aborted.
